### ISSUE TITLE
Force Implant to Use DNS Server

### ISSUE THREAD

**Issue by InitRoot** [BishopFox/sliver#718]
I'm having some issues with the DNS implants constantly timeouts. I've found if I force the dns server with dnscat for example -DNSServer X.X.X.X I don't get any issues.

Is there a way to force my sliver implants to use a specific DNS server?

**Comment by moloch--** [comment]
yes, but it's not user-friendly, you can used advanced options to force a resolv.conf - https://github.com/BishopFox/sliver/wiki/C2-Advanced-Options#dns-c2-advanced-options

Note even on windows it needs to be formatted as a `resolv.conf` file including newlines/etc. in the URL argument.
